mmdet/visualization/palette.py

```diff
@@ -87,7 +87,7 @@ def _get_adaptive_scales(areas: np.ndarray,
     Returns:
         ndarray: The adaotive scales with the shape of (n, ).
     """
-    scales = 0.5 + (areas - min_area) / (max_area - min_area)
+    scales = 0.5 + (areas - min_area) // (max_area - min_area)
```

Comment:
Do I need to change it?

Reply:
if i don't use this, the circle ci will raise error,
![IOgpb96fCD](https://user-images.githubusercontent.com/121282623/227089099-8c5ee198-7b17-48da-8f45-8660cc9c2026.jpg), but only mmdet code can't appear this problem.